### PR TITLE
Mobile Activity Badge

### PR DESCRIPTION
**Pull request by jakzaizzat**
### Summary of Changes

1. Show Top member badge in the profile screen
2. Added new notification - `YouReceivedTopActivityBadgeNotification`

### Demo or Before/After Pics

https://github.com/gallery-so/gallery/assets/4480258/820d0214-276a-45a3-ac3b-d0949486f2d5

### Edge Cases

List any common edge cases that you have considered and tested.

1. User that has more than 1 badge
2. User that only has 1 badge

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [x] I've tested the changes and all tests pass.
- [x] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/Notification/NotificationSkeleton.tsx [hunk 1]
```diff
@@ -218,15 +220,19 @@ export function NotificationSkeleton({
...
+            <ProfilePictureBubblesWithCount
+              eventElementId="Notification Row PFP Bubbles"
+              eventName="Notification Row PFP Bubbles Pressed"
+              eventContext={contexts.Notifications}
+              onPress={handleBubblesPress}
+              userRefs={responsibleUsers}
+              totalCount={responsibleUserRefs.length}
+              size="md"
+            />
+          )}
```
Comment (Robinnnnn) [inline apps/mobile/src/components/Notification/NotificationSkeleton.tsx:235]: nit: `overridePfpElement ?? <ProfilePictureBubblesWithCount ... />`

File: apps/mobile/src/components/Notification/Notifications/YouReceivedTopActivityBadge.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  return (
+    <NotificationSkeleton
+      queryRef={query}
+      onPress={handlePress}
+      responsibleUserRefs={[]}
+      notificationRef={notification}
+      overridePfpElement={
+        <View className="h-8 w-8 rounded-full border border-activeBlue items-center justify-center">
+          <TopMemberBadgeIcon />
+        </View>
```
Comment (Robinnnnn) [inline apps/mobile/src/components/Notification/Notifications/YouReceivedTopActivityBadge.tsx:76]: update badge svg

File: apps/mobile/src/components/ProfileView/ProfileView.tsx [hunk 3]
```diff
@@ -181,28 +185,88 @@ type ProfileViewUsernameProps = {
...
   );
 
+  const isActivityBadgeEnabled = isFeatureEnabled(FeatureFlag.ACTIVITY_BADGE, query);
+
+  const bottomSheetRef = useRef<GalleryBottomSheetModalType | null>(null);
+
+  const filteredBadges = useMemo(() => {
+    const badges = query.userByUsername?.badges ?? [];
+    return badges.filter((badge) => badge?.imageURL);
+  }, [query.userByUsername?.badges]);
```
Comment (Robinnnnn) [inline apps/mobile/src/components/ProfileView/ProfileView.tsx:220]: i think we want to further by `BADGE_ENABLED_COMMUNITY_ADDRESSES`, similar to other usages

File: apps/mobile/src/components/Notification/NotificationSkeleton.tsx (hunk 1 above)
Comment (jakzaizzat) [reply apps/mobile/src/components/Notification/NotificationSkeleton.tsx]: done!

File: apps/mobile/src/components/Notification/Notifications/YouReceivedTopActivityBadge.tsx (hunk 2 above)
Comment (jakzaizzat) [reply apps/mobile/src/components/Notification/Notifications/YouReceivedTopActivityBadge.tsx:76]: updated!

File: apps/mobile/src/components/Notification/Notifications/YouReceivedTopActivityBadge.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      <View className="flex">
+        <Text className="dark:text-white">
+          <Typography
+            font={{
+              family: 'ABCDiatype',
+              weight: 'Regular',
+            }}
+            className="text-sm"
+          >
+            You've earned a new badge as one of the top 60 active members on Gallery!
```
[the remark this answers is not in the document]
Comment (jakzaizzat) [reply apps/mobile/src/components/Notification/Notifications/YouReceivedTopActivityBadge.tsx]: Updated!

File: apps/mobile/src/components/ProfileView/ProfileView.tsx (hunk 3 above)
Comment (jakzaizzat) [reply apps/mobile/src/components/ProfileView/ProfileView.tsx]: Updated the logic just like in web

**Review by Robinnnnn — APPROVED** [review]

**Pull request merged**